library/core/registry/src/main/java/org/quiltmc/qsl/registry/api/event/RegistryMonitor.java

```diff
@@ -47,7 +48,8 @@
 	 * Entries must also match the monitor's filters.
 	 * <p>
 	 * Registration inside the callback must use the {@link RegistryEntryContext#registry()} method to get the registry instance,
-	 * for example: {@code Registry.register(context.registry(), id, block);}.
+	 * or alternatively use the {@link RegistryEntryContext#register(Identifier, Object)} method,
```

Comment:
I would actually suggest this as the primary method, and recommend the non-context method as an alternative